kubernetes/ccloudexporter.dp.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: ccloudexporter
+  name: ccloudexporter
+spec:
+  progressDeadlineSeconds: 600
+  replicas: 1
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      app: ccloudexporter
+  strategy:
+    rollingUpdate:
```

Comment:
Is this part required as we have only one replica? 
(Note: having only one replica is probably the best way to go, as having multiple would create unnecessary load on the Metrics API server)

Reply:
For this example only the following lines are required:

```yaml
  selector:
    matchLabels:
      app: ccloudexporter
```